src/controllers/users.js

```diff
@@ -1,6 +1,14 @@
 /* Get all users from Firebase */
-export const getUsers = (req, res) => {
-  // TODO: Use Firebase functions to get all user objects
+import * as admin from 'firebase-admin';
+
+admin.initializeApp({
+  credential: admin.credential.applicationDefault(),
+});
```

Comment:
I see what's going on! Looks you call `admin.initializeApp` twice in two different files.

You should move this block of code to `server.js` where the rest of the third-party packages get initialized. And then delete the identical section in `authentication.js`
A good spot to place this chunk of code could be right above [this line](https://github.com/ijemmao/igbo_api/blob/master/src/server.js#L19)